lib/SIL/SILInstructions.cpp

```diff
@@ -649,6 +649,8 @@ std::pair<SILValue, SILValue> AutoDiffFunctionInst::
 getAssociatedFunctionPair(unsigned differentiationOrder) const {
   assert(differentiationOrder > 0 &&
          differentiationOrder <= this->differentiationOrder);
+  assert(!getAssociatedFunctions().empty() &&
+         "Associated functions may not exist in raw SIL");
```

Comment:
I don't understand this message. Does raw SIL have anything to do with this?

Reply:
When associated functions are not specified, the differentiation pass will fill them in. When the stage is raw, we cannot assume AD has already run.

Reply:
Clarified.